projects/kit/components/like/like.component.ts

```diff
@@ -0,0 +1,51 @@
+import {
+    ChangeDetectionStrategy,
+    Component,
+    inject,
+    Input,
+    ViewEncapsulation,
+} from '@angular/core';
+import {tuiIsString} from '@taiga-ui/cdk/utils/miscellaneous';
+import {
+    tuiAppearanceOptionsProvider,
+    TuiWithAppearance,
+} from '@taiga-ui/core/directives/appearance';
+import {tuiInjectIconResolver} from '@taiga-ui/core/tokens';
+import type {TuiSizeS} from '@taiga-ui/core/types';
+
+import {TUI_LIKE_OPTIONS, type TuiLikeOptions} from './like.options';
+
+@Component({
+    standalone: true,
+    selector: 'input[tuiLike]',
+    template: '',
+    styleUrls: ['./like.style.less'],
+    encapsulation: ViewEncapsulation.None,
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    providers: [tuiAppearanceOptionsProvider(TUI_LIKE_OPTIONS)],
+    hostDirectives: [TuiWithAppearance],
+    host: {
+        '[attr.data-size]': 'size',
+        '[attr.data-mode]': '""',
+        '[style.--t-icon-color]': 'iconColor',
+        '[style.--t-unchecked-icon]': 'getIcon("unchecked")',
+        '[style.--t-checked-icon]': 'getIcon("checked")',
+    },
+})
+export class TuiLike {
+    private readonly options = inject(TUI_LIKE_OPTIONS);
+    private readonly resolver = tuiInjectIconResolver();
+
+    @Input('tuiLike')
+    public iconColor: string | '' = '';
```

Comment:
Can probably be just `color` for brevity

Reply:
done